pkg/web_app/lib/src/search.dart

```diff
@@ -4,13 +4,44 @@
 
 import 'dart:html';
 
+import 'package:web_app/src/gtag_js.dart';
+
 void setupSearch() {
+  _setEventForKeyboardShortcut();
   _setEventForSearchInput();
   _setEventForFiltersToggle();
   _setEventForSortControl();
   _setEventForCheckboxChanges();
 }
 
+void _setEventForKeyboardShortcut() {
+  final inputElem = document.querySelector('input.site-header-search-input');
+  if (inputElem != null && inputElem is InputElement) {
+    window.onKeyPress.listen((e) {
+      // Ignore keys other than the shortcut key:
+      if (e.key != '/') return;
+
+      // Only trigger the input field and steal focus when nothing is focused,
+      // or when the focused element is not an input element.
+      final active = document.activeElement;
+      final isRestricted = active is InputElement || active is TextAreaElement;
+      if (!isRestricted) {
+        inputElem.focus();
+
+        // prevent the trigger character to get typed into the input field
+        e.preventDefault();
+
+        // notify analytics
+        gtagEvent(
+          'keyboard-shortcut-search',
```

Comment:
I think this could just be `focus-search`.

I'm not sure it matters much.. but ordering seems to be `category > action > label` at least in the "top events" view :D
![AEMqxTfBmS9gFVE](https://user-images.githubusercontent.com/149732/106253964-7b29b480-6218-11eb-9079-88fb23d70ecb.png)